code_schemes/somalia_district.json

```diff
@@ -0,0 +1,884 @@
+{
+  "SchemeID": "Scheme-5292e8fb",
+  "Name": "Somalia District",
```

Comment:
Do you definitely want only the district scheme and not any of the less specific location schemes?

Reply:
Yes we only have one district question.